manifests/1.2.0/opensearch-1.2.0.yml

```diff
@@ -7,6 +7,8 @@ ci:
 build:
   name: OpenSearch
   version: 1.2.0
+  platform: linux
+  architecture: x64
```

Comment:
The input manifest should not be locking the platform and architecture, remove.

Reply:
Is there any other way of doing this because platform and architecture are taken from builds/manifest which is not present in the input manifest https://github.com/opensearch-project/opensearch-build/blob/main/src/jenkins/BuildManifest.groovy#L21

Reply:
Isn't the input to the job a build ID that gives me a URL to a build YAML?